notifications_utils/version_tools.py

```diff
@@ -4,8 +4,12 @@
 
 requirements_file = pathlib.Path("requirements.in")
 frozen_requirements_file = pathlib.Path("requirements.txt")
-pyproject_file = pathlib.Path("pyproject.toml")
 repo_name = "alphagov/notifications-utils"
+config_files = {
+    "pyproject.toml",
+    "requirements_for_test_common.txt",
+    ".pre-commit-config.yaml",
```

Comment:
i think this fine for now looking at our files but this may make it a pain if we want to have specific additional pre commit hooks for specific repos.

eg if we wanted to add some frontend scss linting/formatting stuff to admin